### ISSUE TITLE
[Remove-NavContainer] Cannot bind argument to parameter Path because it is null

### ISSUE THREAD

**Issue by JulienDecottignies** [Microsoft/navcontainerhelper#129]
Get this error message when executing RemoveContainer.

$ExtensionsFolder var is not set before doing the JoinPath to initialize the $containerFolder var.

Fixed by adding parameter **[string] $ExtensionsFolder** in Remove-NavContainer.ps1

**Comment by JulienDecottignies** [comment]
Oh... Sorry I tried just by calling remove-navcontainer in powershell terminal, not by psm1 file. 
Talked to fast..!

**Comment by freddydk** [comment]
Yeah - you need to import / install the module - if you have downloaded the source - you can run the navcontainerhelper.ps1 script in the root.
